### ISSUE TITLE
Negative TC in loss computation

### ISSUE THREAD

**Issue by Greek-Guardian** [ramber1836/d3vae#3]
In the file [https://github.com/zpham/D3VAE/blob/main/exp/exp_model.py](url)
`loss = loss1*self.args.psi + loss2*self.args.lambda1 + mse_loss - self.args.gamma*total_c`
Why is total_c negative while we want it be small?

**Comment by yl4467** [comment]
Since total_c denotes the disentanglement loss, bigger total_c means better disentanglement performance.
